addons/CLib/MapGraphics/fn_mapGraphicsPosition.sqf

```diff
@@ -2,17 +2,17 @@
 /*
     Community Lib - CLib
 
-    Author: BadGuy
+    Author: BadGuy, Raven
 
     Description:
     Converts a Position from MapGraphicsPosition into a position
 
     Parameter(s):
-    0: Position <Array, Object> (Default: objNull)
+    0: Position <Array, Object> (MapGraphicsPosition) (Default: objNull)
```

Comment:
The parameter section should state `Array, Object` instead of `MapGraphicsPosition`

Reply:
It does state `Array, Object` though. If you don't mention that this is supposed to be a `MapGraphicsPosition`, how should the user know what to do with the remark below about what that actually is?

Reply:
this function should never be called by a user it is a function that is only used internaly

Reply:
That doesn't mean that the docs have to be useless

Reply:
if you think so than you need to define MapGraphicsPosition in the remarks.

Reply:
It actually is defined already